keps/sig-api-machinery/1040-priority-and-fairness/README.md

```diff
@@ -2806,9 +2808,66 @@ with miniscule quanta, are also isomorphic.
 
 ### Test Plan
 
-- __Unit Tests__: All changes must be covered by unit tests. Additionally,
- we need to test the evenness of dispatching algorithm.
-- __Integration Tests__: The use cases discussed in this KEP must be covered by integration tests.
+[x] I/we understand the owners of the involved components may require updates to
+existing tests to make this code solid enough prior to committing the changes necessary
+to implement this enhancement.
+
+##### Prerequisite testing updates
+
+Reviewers have not identified any additional tests that need to be
+added prior implementing this enhancement to ensure the enhancements
+have also solid foundations.
+
+##### Unit tests
+
+Integration and end-to-end testing of this feature are difficult
+because it is all about performance and timing but our CI does not run
+the tests on nodes with predictable performance (because they are
+doing many tests at once).  Therefore as much as possible of the
```

Comment:
I'm not sure I would be digging into all those details here - the more text the more question.

Let's stick to the template and just simply list the more critical packages and their current coverage

I think it's mostly:
- staging/src/k8s.io/apiserver/pkg/util/flowcontrol/
- staging/src/k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing
- staging/src/k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/promise
- staging/src/k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/queueset
- staging/src/k8s.io/apiserver/pkg/util/flowcontrol/request
- staging/src/k8s.io/apiserver/pkg/server/filters    # for filter